olp-cpp-sdk-dataservice-read/src/ApiClientLookup.cpp

```diff
@@ -57,7 +57,7 @@ ApiClientLookup::ApiClientResponse ApiClientLookup::LookupApi(
     client::OlpClientSettings settings) {
   repository::ApiCacheRepository repository(catalog, settings.cache);
 
-  if (options != OnlineOnly) {
+  if (options != OnlineOnly && options != CacheWithUpdate) {
```

Comment:
Why also CacheWithUpdate?

Reply:
Previously for CacheWithUpdate was splitted to CacheOnly and OnlineOnly. Now with OnlineOnly we are not updating cache. So for updating cache and getting data only online, even if we have valid stored cache we need additional fetch option. And to not brake api, I used CacheWithUpdate for this case.